New+ Feature - Doc Update - Option to hide the built-in New context m…

cgaarden opened the pull request:
PowerToys PR: 
[x] [New+] Hide existing new - remake #44979

Scope of changes
* Updated PowerToys New+ documentation to cover the latest feature to enable users to hide the built-in File Explorer New context menu

cgaarden committed:
New+ Feature - Doc Update - Option to hide the built-in New context menu (37545 and 37946)

Doc Update - Option to hide the built-in New context menu (37545 and 37946)

alvinashcraft commented on hub/powertoys/newplus.md:
```diff
@@ -47,6 +47,11 @@ The option allows you to toggle the display of filename extensions. When this op
 
+#### Hide the built-in New context menu
+
+With this option you can hide the File Explorer built-in New content menu.
```
```suggestion
With this option, you can hide the File Explorer built-in New content menu.

```

alvinashcraft commented on hub/powertoys/newplus.md:
```diff
@@ -47,6 +47,11 @@ The option allows you to toggle the display of filename extensions. When this op
+#### Hide the built-in New context menu
+
+With this option you can hide the File Explorer built-in New content menu.
+Note: You will need to run PowerToys as an Administrator to toggle this option. Also note, as a precaution, the built-in New will automatically be turned-back-on during PowerToys uninstall.
```
Do you want this to be formatted as a Note alert box? If not, I would still suggest using the minor wording updates within this suggestion.
```suggestion
> [!NOTE]
> You need to run PowerToys as an Administrator to toggle this option. Also note that, as a precaution, the built-in New menu will automatically be re-enabled during PowerToys uninstall.
```

alvinashcraft reviewed (commented):
I left some minor suggestions. Thanks for the updates! I'll hold onto this until this feature is released as part of a PowerToys release.

cgaarden committed:
Update hub/powertoys/newplus.md

Co-authored-by: Alvin Ashcraft <73072+alvinashcraft@users.noreply.github.com>

cgaarden committed:
Update hub/powertoys/newplus.md

Co-authored-by: Alvin Ashcraft <73072+alvinashcraft@users.noreply.github.com>

alvinashcraft reviewed (approved):
Thanks again! I'll keep an eye out for this feature's release.

cgaarden committed:
Merge branch 'MicrosoftDocs:docs' into docs

cgaarden committed:
Updated New+ documentation to include "Hide the built-in 'New' context menu"

Also included how to manually update the registry to restore the built-in New context menu

cgaarden reviewed (commented):
Updated to reflect latest and just merged changes in PowerToys

cc: @niels9001
